walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -152,16 +154,16 @@ public void nestedEvents() {
     // Have an event for each person, but have one person's event fully contain another's event. We
     // should see two options.
     //
-    // Events  :       |----A----|
-    //                   |--B--|
+    // Events  :      |--B--|       
+    //              |----A----|
     // Day     : |---------------------|
     // Options : |--1--|         |--2--|
 
     Collection<Event> events = Arrays.asList(
-        new Event("Event 1", TimeRange.fromStartDuration(TIME_0830AM, DURATION_90_MINUTES),
-            Arrays.asList(PERSON_A)),
         new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
-            Arrays.asList(PERSON_B)));
+            Arrays.asList(PERSON_B)),
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0830AM, DURATION_90_MINUTES),
+            Arrays.asList(PERSON_A)));
```

Comment:
You should not make changes to existing tests (even thought, this change seems to just swap the two time slots).

Reply:
Umm I just wanted to test this case out, cause the original case was working fine but when I tried to swap the events, I realised my logic wasn't complete, so was just trying to use it.

Reply:
I will revert it back to the original test.

Reply:
And add it as a new one.